core/server/api/canary/utils/serializers/output/mappers/posts.js

```diff
@@ -57,8 +61,6 @@ module.exports = async (model, frame, options = {}) => {
         }
         date.forPost(jsonModel);
         gating.forPost(jsonModel, frame);
-
-        delete jsonModel.newsletter_id;
```

Comment:
Note that this is dropped because this is removed a couple of lines later in `clean.post(jsonModel, frame);` in both the content and admin api